src/amo/pages/AddonInfo/styles.scss

```diff
@@ -18,3 +18,15 @@
     width: 65%;
   }
 }
+
+.AddonMoreInfo-tag-links-list > li {
+  display: inline-block;
+}
+
+.AddonMoreInfo-tag-links-list > li::after {
+  content: ',\00a0';
+}
+
+.AddonMoreInfo-tag-links-list > li:last-child::after {
```

Comment:
This isn't working for RTL mode:

![Screenshot 2021-07-28 at 10 36 34](https://user-images.githubusercontent.com/217628/127291700-cdc85dd1-f895-424c-8e3f-98c606097338.png)

Reply:
https://github.com/mozilla/addons-frontend/pull/10798/commits/106d2012ebd33ddf65876649ae6f72df692996d3

looks like this now:
![rtl Screenshot 2021-07-28 133509](https://user-images.githubusercontent.com/768592/127323804-48d187bc-105a-48b5-bc53-2ee41736a270.png)

and LTR:
![Screenshot 2021-07-28 133414](https://user-images.githubusercontent.com/768592/127323848-86c892cc-7afc-426a-8bc8-4928f87d0a13.png)

Reply:
@jvillalobos is a bullet point okay rather than a comma?  It works more consistently in RTL (and other non-Latin languages that use different characters)

Reply:
or this is with a smaller bullet point character
![Screenshot 2021-07-28 144003](https://user-images.githubusercontent.com/768592/127332388-99f99b3d-f6b7-4c6c-95d7-0c1383e3fa63.png)

Reply:
The small bullet point looks good to me.